enterprise/frontend/src/metabase-enterprise/content_verification/components/VerifiedSearchFilter/VerifiedSearchFilter.styled.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+import { color, lighten } from "metabase/lib/colors";
+
+export const VerifiedFilterButton = styled(Button)<{ isSelected: boolean }>`
+  ${({ isSelected }) =>
+    isSelected
+      ? `
```

Comment:
nit: using `css` from `@emotion/react` would make DX viewing the CSS rules better for supported IDEs. For example, these screenshots are from my setup (VS Code)

#### This PR
![image](https://github.com/metabase/metabase/assets/1937582/f00c5bc9-1bab-45f6-8e1f-8b2b95050822)

#### With `css` it's formatted like CSS
![image](https://github.com/metabase/metabase/assets/1937582/a6933a55-6814-4e9c-9956-b1d802dc6ee2)